mvg/utils/__init__.py

```diff
@@ -0,0 +1,2 @@
+def clamp(value, minimum, maximum):
+    return max(minimum, min(value, maximum))
```

Comment:
this is not a initialization method, so i do not think this file method should be here